docs/network-iot/run-an-lns/buy-an-oui.mdx

```diff
@@ -107,6 +107,9 @@ files accordingly.
 **Delegate Key Pair**: The delegate key is actively used to create and manage routes. In case it is
 ever compromised, it can be updated using the owner key.
 
+- Note that you can have multiple delegate key pairs, you can see how to do that
```

Comment:
Trying to avoid "you". Would recommend "an OUI can have..."

Links like "here" are not very clear. Should aim for "Guidance for adding multiple ... are in [advanced configuration]"